src/FormBuilder/ViewModels/FormBuilderViewModel.php

```diff
@@ -115,4 +121,26 @@ public function getGateways(): array
 
         return array_values($builderPaymentGatewayData);
     }
+
+    /**
+     * @unreleased
+     */
```

Comment:
Please add a small description explaining what this is for. This is really helpful.

Reply:
Resolved in c1b410c8